### PR TITLE
Cheat sheet, assembly changes and infrastructure changes

### PR DESCRIPTION
**Pull request by janajaeger**
After our Smart Docs template training, a few things come to our minds:

- snippet handling needs to be added (dir, example, assembly reference)
- trans and bugtracker meta tags aren't supported yet
- we need a cheat sheet for the writers ...

### CODE REVIEW

File: templates/articles/assembly.asm.xml [hunk 1]
```diff
@@ -87,12 +87,16 @@
...
       <meta name="updated" content="2023-11-11"/>
-      <meta name="bugtracker"><phrase role="url">https://bugzilla.suse.com/enter_bug.cgi</phrase><phrase role="component">Non-product-specific documentation</phrase><phrase role="product">Smart Docs</phrase><phrase role="assignee">assignee@suse.com</phrase>
-      </meta>
-      <meta name="translation">
-        <!-- enter yes or no for translation --><phrase role="trans">yes</phrase>
-        <!-- enter languages: ar-ar, cs-cz, ... --><phrase role="language">comma-separated list of languages, for example de-de,cs-cz</phrase>
-      </meta>
+      <!-- this does not work yet. Use the dm tags listed below for now
+        <meta name="bugtracker"><phrase role="url">https://bugzilla.suse.com/enter_bug.cgi</phrase><phrase role="component">Non-product-specific documentation</phrase><phrase role="product">Smart Docs</phrase><phrase role="assignee">assignee@suse.com</phrase>
+        </meta>
```
Comment (tomschr) [inline templates/articles/assembly.asm.xml:92]: I think, you can format it more nicely:

```suggestion
        <meta name="bugtracker">
          <phrase role="url">https://bugzilla.suse.com/enter_bug.cgi</phrase>
          <phrase role="component">Non-product-specific documentation</phrase>
          <phrase role="product">Smart Docs</phrase>
          <phrase role="assignee">assignee@suse.com</phrase>
        </meta>
```

File: templates/articles/assembly.asm.xml [hunk 2]
```diff
@@ -104,6 +108,17 @@
...
       <meta name="category" content="Systems Management"/>
+      <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+        <dm:bugtracker>
+          <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+          <dm:component>Smart Docs</dm:component>
+          <dm:product>Documentation</dm:product>
+          <dm:assignee>maintainer@suse.com</dm:assignee>
+        </dm:bugtracker>
+        <dm:editurl>https://github.com/SUSE/doc-modular/tree/main/articles/
+      </dm:editurl>
```
Comment (tomschr) [inline templates/articles/assembly.asm.xml:119]: Linebreaks are... evil. :wink: 

```suggestion
        <dm:editurl>https://github.com/SUSE/doc-modular/tree/main/articles/</dm:editurl>
```

File: templates/common/legal.xml
Comment (tomschr) [inline templates/common/legal.xml:1]: If you move the snippet dir, this needs to be adapted.

File: templates/snippets/snippet_warning.xml
Comment (tomschr) [inline templates/snippets/snippet_warning.xml:1]: Same here.

**Review by tomschr — CHANGES_REQUESTED** [review]
@janajaeger Some minor issues. :slightly_smiling_face:

File: snippets/snippet_warning.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE warning
+[
+  <!ENTITY % entities SYSTEM "../common/generic-entities.ent">
+    %entities;
+]>
+<warning xml:id="foobar"
```
[the remark this answers is not in the document]
Comment (janajaeger) [reply snippets/snippet_warning.xml]: Moved the snippet itself and will remove the ID there.

File: templates/snippets/snippet_warning.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+<!DOCTYPE warning
+[
+  <!ENTITY % entities SYSTEM "../common/generic-entities.ent">
+    %entities;
+]>
+<warning
+  version="5.2" xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>Foobar</titel>
```
Comment (dariavladykina) [inline templates/snippets/snippet_warning.xml:11]:
```suggestion
  <title>Foobar</title>
```

File: templates/articles/assembly.asm.xml [hunk 5]
```diff
@@ -104,6 +112,16 @@
       <meta name="social-descr">ultra short description for social media, max 55 chars</meta>
```
Comment (dariavladykina) [inline templates/articles/assembly.asm.xml:112]:
```suggestion
      <meta name="social-descr">ultrashort description for social media, max 55 chars</meta>
```

**Review by dariavladykina — COMMENTED** [review]
Hi, just minor nits detected. Thanks!

**Pull request merged**